data/org.freedesktop.impl.portal.ScreenCast.xml

```diff
@@ -123,6 +154,37 @@
               'multiple' was set to 'true' as part of the SelectSources method.
             </para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>persist_mode u</term>
+            <listitem><para>
+              Portal implementations can optionally offer ways to "reduce" the
+              persist mode, for example by explicitly asking the user. See the
+              'persist_mode' option of SelectSources for a list of possible
+              values.
+
+              If no persist mode is returned by the Start request, it is assumed
+              to be the same persist mode received during SelectSources.
+
+              This option was added in version 4 of this interface.
+            </para></listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>restore_data v</term>
+            <listitem><para>
+              The data to be restored. This is the data that a portal
+              implementation sent in reaction to transient or persistent
+              modes.
+
+              Data generated by a portal implementation must be consumable by
+              the same portal implementation, but not necessarily by others.
+              Thus, it is important to properly handle 'restore_data' not being
+              in a valid format. This may happen when, e.g., the user switched
+              to another desktop environment, or is using a different portal
+              implementation.
```

Comment:
I don't think the permission store has any particular requirements for the variant. I opted for  a simple `v` here because that's as generic as it can be, and can cover pretty much everything any portal implementation need.

Reply:
Thanks for the suggestion. I've added a paragraph with (a modified version of) it.